src/main/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceInputFormat.java

```diff
@@ -212,7 +220,15 @@ private BulkConnection getBulkConnection(Configuration conf) {
         LOG.info(String.format("Job with Id: '%s' is aborted", jobId));
         return new BatchInfo[0];
       }
-      initialBatchInfo = bulkConnection.getBatchInfo(jobId, initialBatchId);
+      try {
+        initialBatchInfo = bulkConnection.getBatchInfo(jobId, initialBatchId);
+      } catch (AsyncApiException e) {
+        if (i == SalesforceSourceConstants.GET_BATCH_RESULTS_TRIES - 1) {
+          throw e;
+        }
+        LOG.warn("Failed to get batch info: {}, retry {}", e, i);
```

Comment:
similar comment here, exceptions should be the last element in a log statement, and should not have a corresponding placeholder `{}`.